dotnet/secrethub.csproj

```diff
@@ -1,13 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Why put both versions of `netstandard` and remove the `netcoreapp3.1`?

Reply:
Discussed in person and sounds like the right choice.